src/main/java/com/redhat/exhort/providers/GradleProvider.java

```diff
@@ -253,7 +253,7 @@ private Sbom buildSbomFromTextFormat(
 
     for (String line : lines) {
       line = line.replaceAll("---", "-").replaceAll("    ", "  ");
-      line = line.replaceAll(":(.*):(.*) -> (.*)$", ":$1:$3");
+      line = line.replaceAll(":(.*):(.*) -> (.*)$", ":$1:$2");
```

Comment:
@olavtar This is indeed make the stack analysis consistent with component analysis. :)
but unfortunately it's incorrect 
Because the jar package will be eventually with the resolved version, and not the stated version in the manifest.
the solution is another way around - to fetch all component analysis artifacts from their configurations, and then lookup the right version from stack analysis resolved version.

Reply:
Done

Reply:
@olavtar Now it fails on integration tests because of HTML structure returned from exhort changed recently...
 I Think since html and its embedded json report structure changed from time to time, it's more correct to test the html report of the integration tests just with checking that the returned text is indeed html format ( html/text format) and that's it,  rather than in addition trying to parse the embedded json report inside the big html document, because this might change every now and then.

Reply:
@zvigrinberg I'm not aware of any structural changes to the report. Could you please point out what has changed?